src/components/number-input/number-input.ts

```diff
@@ -123,6 +152,12 @@ export default class BXNumberInput extends BXInput {
   @property({ attribute: 'decrement-button-assistive-text' })
   decrementButtonAssistiveText = 'decrease number input';
 
+  @property({ attribute: 'validity-message-max' })
```

Comment:
If possible, could you add comments to the two new attributes? Doing so makes the comment appear in the property table in Docs page in Storybook.

Reply:
For sure!